layouts/shortcodes/config-file-istio-dex.html

```diff
@@ -1 +1 @@
-kfctl_istio_dex.v1.0.2.yaml
\ No newline at end of file
+kfctl_istio_dex.yaml
```

Comment:
@swiftdiaries should this be kfctl_istio_dex.v1.1.0.yaml?

Reply:
Addressed 🙌🏾